packages/fxa-settings/src/components/Ready/index.stories.tsx

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import Ready from '.';
+import { ViewNameType } from '.';
+import AppLayout from '../../components/AppLayout';
+import { Meta } from '@storybook/react';
+
+export default {
+  title: 'pages/Ready',
```

Comment:
This should probably go under `components/Ready` instead of `pages` to match the pattern of where it in our directory structure.

Reply:
oh smart of course, definitely

Reply:
done!